turborepo-tests/integration/tests/link.t

```diff
@@ -1,5 +1,5 @@
 Setup
-  $ . ${TESTDIR}/_helpers/setup.sh
+  $ . ${TESTDIR}/../../helpers/setup.sh
```

Comment:
I'd really like for this to be:
`node -r path -e 'console.log(path.dirname(require.resolve("turborepo-tests-helpers/package")));'`

...but we'd need to make that "run-once" instead of "run a billion times".

Reply:
yeah, we could at least set `MONOREPO_ROOT_DIR`, instead of relying on prysk's TESTDIR directly, here, but honestly, don't want to do another round on this PR right now heh. It would just be an aesthetic change

Reply:
Yeah. This isn't blocking.